src/ComposerRequireChecker/Cli/Options.php

```diff
@@ -36,8 +36,16 @@ class Options
         'never',
     ];
 
+    private const PHP_ATTRIBUTES = [
+        'AllowDynamicProperties',
+        'Attribute',
+        'Override',
+        'ReturnTypeWillChange',
+        'SensitiveParameter',
```

Comment:
Let's use `::class` references for these, please :)

Reply:
yes, it is fixed